pkg/cronjob/spark_dependencies_test.go

```diff
@@ -79,6 +79,17 @@ func TestCreate(t *testing.T) {
 	assert.NotNil(t, CreateSparkDependencies(&v1.Jaeger{Spec: v1.JaegerSpec{Storage: v1.JaegerStorageSpec{Type: "elasticsearch"}}}))
 }
 
+func TestSparkDependenciesSecretSecrets(t *testing.T) {
```

Comment:
nit: `TestSparkDependenciesSecrets` ? - if changed can you also update the test name in the line below.

Reply:
Good suggestion. It is changed now.